backend/migrations/20230703130438_food.up.sql

```diff
@@ -1,5 +1,5 @@
 -- Add up migration script here
-CREATE TYPE meal_type AS ENUM ('VEGAN','VEGETARIAN','BEEF','BEEF_AW','PORK','PORK_AW','FISH','UNKNOWN');
```

Comment:
As already stated last time, you are not allowed to edit existing, already deployed migrations. Rather create a new migration where you edit the enum.

Reply:
In this new migration, you should also modify all existing meals where the poultry regex matches the new meal type, so new parsed meals can get merged with the existing ones.